editor/import/scene_import_settings.cpp

```diff
@@ -1598,6 +1606,14 @@ SceneImportSettings::SceneImportSettings() {
 		collider_mat->set_albedo(Color(0.5, 0.5, 1.0));
 	}
 
+	{
+		bones_mesh_preview = memnew(MeshInstance3D);
+		bones_mesh_preview->set_cast_shadows_setting(GeometryInstance3D::SHADOW_CASTING_SETTING_OFF);
+		bones_mesh_preview->set_skeleton_path(NodePath(""));
+		bones_mesh_preview->translate(Vector3(0.0, 0.0, 0.035));
```

Comment:
This seems to be a mistake.
![godot windows editor dev x86_64_n6ulT8u9mz](https://github.com/godotengine/godot/assets/12756047/26d255fa-e392-4c6f-b553-eae8046c7897)